packages/cloud-toolkit/src/templates/.env.txt

```
@@ -0,0 +1,2 @@
+FOREST_ENV_SECRET=FOREST_ENV_SECRET_TO_REPLACE
```

```suggestion
FOREST_ENV_SECRET=<FOREST_ENV_SECRET_TO_REPLACE>
```
You could have `<>` surrounding replace patterns